docs/data-cloud/aspire-integration.md

```diff
@@ -0,0 +1,344 @@
+
+### Create a service client
+
+Create a typed client to consume your web service:
+
+```csharp
+public class WeatherApiClient
+{
+    private readonly HttpClient _httpClient;
+
+    public WeatherApiClient(HttpClient httpClient)
+    {
+        _httpClient = httpClient;
+    }
+
+    public async Task<WeatherForecast[]?> GetWeatherForecastAsync(CancellationToken cancellationToken = default)
+    {
+        return await _httpClient.GetFromJsonAsync<WeatherForecast[]>(
+            "/weatherforecast",
+            cancellationToken);
+    }
+}
+
+public record WeatherForecast(DateOnly Date, int TemperatureC, string? Summary)
+{
+    public int TemperatureF => 32 + (int)(TemperatureC / 0.5556);
+}
+```
+
+### Use the client in your app
+
+Inject and use the client in your pages or view models:
+
+```csharp
+public partial class MainPage : ContentPage
+{
+    private readonly WeatherApiClient _weatherClient;
+
+    public MainPage(WeatherApiClient weatherClient)
+    {
+        _weatherClient = weatherClient;
+        InitializeComponent();
+    }
+
+    private async void OnGetWeatherClicked(object sender, EventArgs e)
+    {
+        try
+        {
+            var forecasts = await _weatherClient.GetWeatherForecastAsync();
+            
+            if (forecasts != null)
+            {
+                // Display the weather data
+                ResultLabel.Text = $"Retrieved {forecasts.Length} forecasts";
+            }
+        }
+        catch (Exception ex)
+        {
+            ResultLabel.Text = $"Error: {ex.Message}";
+        }
+    }
+}
+```
+
+## Platform-specific considerations
+
+### iOS and Mac Catalyst
+
+On iOS and Mac Catalyst, the Aspire integration works seamlessly when running the app through the App Host. The service discovery configuration automatically provides the correct URLs for connecting to local services.
+
+When using the iOS Simulator or running on a physical device, Dev Tunnels are automatically configured to enable connectivity to services running on your development machine.
+
+> [!NOTE]
+> Screenshot suggestion: Show the Aspire dashboard with a MAUI iOS app connected to a web service, displaying the active connections and logs.
+
+### Android
+
+On Android, the Aspire integration handles the platform-specific networking requirements automatically. You no longer need to:
+
+- Configure the special `10.0.2.2` address
+- Set up network security configuration files
+- Enable clear-text traffic for local development
+
+The integration uses Dev Tunnels to provide secure, reliable connectivity between the Android emulator and your local services.
+
+> [!NOTE]
+> Screenshot suggestion: Show the Android emulator running a MAUI app successfully connecting to a local service through the Aspire integration.
+
+#### Dev Tunnels integration
```

Comment:
Make special mention of  the `WithOtlpTunnel()` method as seen here: https://github.com/dotnet/aspire/blob/cf7bd8dbf7e7413077940f074a239307c43b7278/playground/AspireWithMaui/AspireWithMaui.AppHost/AppHost.cs

This is needed to get the OpenTelemetry data from the iOS and Android apps. Not sure if this is the right section, so see where to mention it, but it should be included.